docs/src/content/docs/cookbook/nfts.mdx

```diff
@@ -323,6 +323,86 @@ contract Example {
 }
 ```
 
+## Onchain metadata creation
+
+### NFT Collection {#onchain-metadata-nft-collection}
+
+```tact
+/// https://github.com/ton-blockchain/TEPs/blob/master/text/0064-token-data-standard.md#nft-metadata-attributes
+fun composeCollectionMetadata(
+    name: String,        // full name
+    description: String, // text description of the NFT
+    image: String,       // link to the image
+    // There could be other attributes, see:
+    // https://github.com/ton-blockchain/TEPs/blob/master/text/0064-token-data-standard.md#nft-metadata-attributes
```

Comment:
I would also add this link https://github.com/getgems-io/nft-contracts/blob/main/docs/metadata.md

However onchain NFT attributes are not supported by getgems.